### DISCUSSION TITLE
Add support for installing nchat using homebrew

### DISCUSSION THREAD

**Discussion by Bellavene** [d99kris/nchat#91] [Ideas]
As title says, I believe it is the time to add it to brew...

**Comment by d99kris** [comment]
I can see how that would be useful. Last time I looked into packaging some of my projects (I believe I was looking into Debian or Ubuntu) it seemed the process for doing so was a little involved. I can look into and see if packaging for homebrew is perhaps a little less work. If someone like to help out, any help is welcome 🙂

**Reply by HawkingRadiation42** [reply]
hello @d99kris love to help to add nchat to brew, i have made a pr regarding this do give me a feedback if you have in mind. i have created using tap currently as the homebrew requires the original author to push into brew core.
thankyou

**Comment by d99kris** [comment]
A homebrew tap for nchat has been created now, thanks to @HawkingRadiation42 for the contribution! 
The README has been updated, but in short:
```
brew tap d99kris/nchat
brew install nchat
```
Note 1: This brew tap still builds nchat on your machine, so it takes a little while.

Note 2: You probably want to remove nchat manually built and installed from sources before getting via brew, the simplest way to do so is to clone nchat and then do:
```
mkdir -p build
cd build
cmake ..
make uninstall
```
